src/server/lib/queryParams.ts

```diff
@@ -33,6 +33,7 @@ export const parseExpressQueryParams = (
     componentEventParams,
     fromURI,
     appClientId,
+    consentToken,
```

Comment:
does the `consentToken` need to be here, and below?

I don't see it being used as a query parameter anywhere